src/vcpkg/postbuildlint.cpp

```diff
@@ -346,10 +353,16 @@ namespace vcpkg::PostBuildLint
     {
         const auto packages_dir = paths.packages() / spec.dir();
         const auto copyright_file = packages_dir / "share" / spec.name() / "copyright";
-        if (fs.exists(copyright_file, IgnoreErrors{}))
+        if (fs.is_regular_file(copyright_file))
```

Comment:
As an aside, almost all places we use `exists` probably should be doing `is_regular_file` or `is_directory` 🤷. Thanks!

It would be good to combine these with `fs.status()` to avoid hitting the disk twice.

Reply:
@BillyONeal the function `check_for_copyright_file` wants to suggest possible copyright files. Do we actually need this functionality? IMO this is not really needed.

Reply:
> As an aside, almost all places we use `exists` probably should be doing `is_regular_file` or `is_directory` 🤷. Thanks!

I'll open another PR for that because this one will focus more on the copyright thing. However, this will take some time because first I have to fix ports that are abusing this and create `vcpkg_install_copyright()`.

Reply:
> @BillyONeal the function `check_for_copyright_file` wants to suggest possible copyright files. Do we actually need this functionality? IMO this is not really needed.

I'm not sure I understand the question. If you mean "do we actually want diagnostics to help people fix our warnings", then yes, we do want such functionality.